iot/manager/resources/rsa_private.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
The attach surface for clients connecting using these credentials is low at present --  a client being able to connect to our registries could read / write data specific to that registry but we remove the registries after the tests finish. That said, it could make sense to pull this from an encrypted bucket -- do you have an example of another sample doing this?

Reply:
Would add the file to the cloud-devrel-kokoro-resource bucket at /go/somepath then add another line like this: https://github.com/GoogleCloudPlatform/golang-samples/blob/master/testing/kokoro/common.cfg#L4. Then, set an env var in https://github.com/GoogleCloudPlatform/golang-samples/blob/master/testing/kokoro/system_tests.sh and read it from these tests to get the file path.